src/applicants/applicantsController.js

```diff
@@ -0,0 +1,21 @@
+import Applicant from "./applicantsModel";
+
+const LIMIT = 8;
```

Comment:
maybe to save limit on constants? just a suggestion

Reply:
fixed